docs/manual/billing_groups.md

```diff
@@ -0,0 +1,64 @@
+# Billing groups
+
+> **Note: ** Billing groups is optional feature.
+
+Billing groups allow Squest administrator to visualize who is consuming what.
+
+## Billing groups management
+
+The administrator can create a new billing group and manage users in a billing groups.
+
+From the sidebar, click on 'Billing Groups'. 
+All billing groups are listed in this page, you can see the name and the users count of each billing group.
+
+### Create a new billing group
+
+From the billing groups page, click on 'Add' button in the upper left corner then choose an unused billing group name.
+
+### Edit a billing group
+
+From the billing groups page, click on 'Edit' button of the billing group then choose an unused billing group name.
```

Comment:
An unused? When selecting existing one?

I think Create/edit/list could be placed in the same section as the UI is clear enough to understand what we can do